pkg/cmd/serve.go

```diff
@@ -32,9 +33,16 @@ func newServeCmd() *serveCmd {
 				dir = args[0]
 			}
 
+			absoluteDir, err := filepath.Abs(dir)
+			if err != nil {
+				return err
+			}
+
+			fmt.Printf("Starting server for directory  %s\n", absoluteDir)
+
 			fmt.Println("Starting static file server at address", fmt.Sprintf("http://localhost:%s", port))
-			http.Handle("/", http.FileServer(http.Dir(dir)))
-			err := http.ListenAndServe(fmt.Sprintf(":%s", port), handlers.LoggingHandler(os.Stdout, http.DefaultServeMux))
+			http.Handle("/", http.FileServer(http.Dir(absoluteDir)))
+			err = http.ListenAndServe(fmt.Sprintf(":%s", port), handlers.LoggingHandler(os.Stdout, http.DefaultServeMux))
 
 			return err
```

Comment:
Since you had to change this line anyway, could be a good opportunity to update to:

```suggestion
			return http.ListenAndServe(fmt.Sprintf(":%s", port), handlers.LoggingHandler(os.Stdout, http.DefaultServeMux))
```